lib/src/components/NxModal/NxModal.scss

```diff
@@ -34,21 +34,12 @@ $nx-modal-content-max-height: calc(100vh - #{$nx-modal-top-spacing} - #{$nx-moda
 
 .nx-modal-footer {
   @include container-vertical;
-  background-color: $nx-grey-1;
-  border-top: $nx-border-dark;
-  padding: 12px 8px;
-
-  &.nx-error {
-    background-color: $nx-fail-color-bg;
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-    margin-right: -1px;
-    margin-bottom: -1px;
-    margin-left: -1px;
+  background-color: #fff;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-l $nx-spacing-xl $nx-spacing-l $nx-spacing-l;
 
-    > .nx-alert--error {
-      border: none;
-    }
+  .nx-alert {
+    margin-bottom: $nx-spacing-xl;
```

Comment:
Is this size of bottom margin supposed to apply only here or to all nx-alerts?

Reply:
> Is this size of bottom margin supposed to apply only here or to all nx-alerts?

I assumed not, I took the values from the Modal Error Zeplin file

Reply:
> I assumed not, I took the values from the Modal Error Zeplin file

The individual component mockups don't really convey external spacing, so I don't think we've had any guidance here one way or the other before this.  @dturner32 ?